specification/communication/data-plane/CallAutomation/preview/2023-01-15-preview/communicationservicescallautomation.json

```diff
@@ -1454,80 +1454,100 @@
         }
       }
     },
-    "ContinuousDtmfRecognitionToneFailed": {
-      "type": "object",
-      "properties": {
-        "toneInfo": {
-          "$ref": "#/definitions/ToneInfo",
-          "description": "Information about Tone."
-        },
-        "callConnectionId": {
-          "description": "Call connection ID.",
-          "type": "string"
-        },
-        "serverCallId": {
-          "description": "Server call ID.",
-          "type": "string"
-        },
-        "correlationId": {
-          "description": "Correlation ID for event to call correlation. Also called ChainId for skype chain ID.",
-          "type": "string"
-        },
-        "resultInformation": {
-          "$ref": "#/definitions/ResultInformation",
-          "description": "Contains the resulting SIP code/sub-code and message from NGC services."
-        }
-      }
-    },
-    "ToneInfo": {
-      "description": "The information about the tone.",
-      "required": [
-        "sequenceId",
-        "tone",
-        "participantId"
-      ],
-      "type": "object",
-      "properties": {
-        "sequenceId": {
-          "format": "int32",
-          "description": "The sequence id which can be used to determine if the same tone was played multiple times or if any tones were missed.",
-          "type": "integer"
-        },
-        "tone": {
-          "$ref": "#/definitions/Tone"
-        },
-        "participantId": {
-          "description": "The id of participant.",
-          "type": "string"
-        }
-      }
-    },
-    "ContinuousDtmfRecognitionToneReceived": {
-      "type": "object",
-      "properties": {
-        "toneInfo": {
-          "$ref": "#/definitions/ToneInfo",
-          "description": "Information about Tone."
-        },
-        "callConnectionId": {
-          "description": "Call connection ID.",
-          "type": "string"
-        },
-        "serverCallId": {
-          "description": "Server call ID.",
-          "type": "string"
-        },
-        "correlationId": {
-          "description": "Correlation ID for event to call correlation. Also called ChainId or skype chain ID.",
-          "type": "string"
-        },
-        "resultInformation": {
-          "$ref": "#/definitions/ResultInformation",
-          "description": "Contains the resulting SIP code/sub-code and message from NGC services."
-        }
-      }
-    },
-    "CommunicationErrorResponse": {
+	"ContinuousDtmfRecognitionToneFailed":{
```

Comment:
why there is difference in space between : and {, some has space some not?!!!

Reply:
It seems the json formatter I used, doesn't add space after colon, though I have fixed now...thanks!